cpu-bind=MASK - bp1-compute007, task  0  0 [1078097]: mask 0x3fff003fff set
Traceback (most recent call last):
  File "/user/home/zj22662/photon-net/reup_vary_new.py", line 183, in <module>
    data_prod_iterator(varied_list, reupload_list, log_file, folder, file_indent, start_idx)
  File "/user/home/zj22662/photon-net/reup_vary_new.py", line 148, in data_prod_iterator
    initial_loss, (n0, key) = loss.loss(
                              ^^^^^^^^^^
TypeError: loss() missing 1 required positional argument: 'use_input_superposition'
--------------------
For simplicity, JAX has removed its internal frames from the traceback of the following exception. Set JAX_TRACEBACK_FILTERING=off to include these.
